dronegen/relcli.go

```diff
@@ -72,10 +83,11 @@ func executeRelcliStep(name string, command string) step {
 			"RELCLI_CERT":     {raw: "/tmpfs/creds/releases.crt"},
 			"RELCLI_KEY":      {raw: "/tmpfs/creds/releases.key"},
 		},
-		Volumes: dockerVolumeRefs(volumeRef{
-			Name: "tmpfs",
-			Path: "/tmpfs",
-		}),
+		Volumes: []volumeRef{
```

Comment:
Seems like `dockerVolumeRefs(volumeRefTmpfs, volumeRefAwsConfig)` would be equivalent.

Reply:
I'll address this when I take care of the discussion at https://github.com/gravitational/teleport/pull/17201#discussion_r991502939, since it is the same idea.